schema3.sql

```diff
@@ -0,0 +1,18 @@
+-- Ping Pong Database
+
+-- Users Table
+CREATE TABLE users (
+  id serial primary key,
+  name text,
+  gender integer,
+  age integer
+);
+
+
+-- Games Table
+CREATE TABLE games (
+  game text,
+  score integer,
+  winning_matches integer,
+  user_id integer references users (id)
```

Comment:
Maybe also have a second foreign key to the `users` table, so that you know who the other player is.